common-libs/common/src/services/DidDocumentService/ElemAnchoredDidDocument.ts

```diff
@@ -0,0 +1,26 @@
+import KeysService from '../KeysService'
+import { DidResolver } from '../../shared/DidResolver'
+
+export default class ElemDidDocument {
```

Comment:
Shouldn't it be `ElemAnchoredDidDocument`?

(Also we probably need to configure an eslint rule for that...)